test/system/065-cp.bats

```diff
@@ -924,20 +926,16 @@ ${randomcontent[1]}" "$description"
 
     # Copy file.
     $PODMAN cp cpcontainer:/tmp/file.txt - > $srcdir/stdout.tar
-    if [ $? -ne 0 ]; then
-        die "Command failed: podman cp ... - | cat"
-    fi
 
     tar xvf $srcdir/stdout.tar -C $srcdir
-    is "$(< $srcdir/file.txt)" "$rand_content"
-    run 1 ls $srcdir/empty.txt
```

Comment:
Completely brokey: `run` is regular bats, not my `run_podman` extension. It does not do the exit-status check. This was actually running the `1` command, which does not exist, and therefore it was failing, but nobody was checking status.